deploy/helm-chart/values.yaml

```diff
@@ -58,6 +58,9 @@ nginx:
     tag: edge
     pullPolicy: Always
 
+  ## Is NGINX Plus image being used
+  isPlus: false
```

Comment:
I'd shorten it to `Plus`. otherwise, we might need to add `is` to any new boolean value in the future for consistency